src/book/garden-circle/contributing.md

```diff
@@ -137,6 +137,27 @@ These workflows include one or more [jobs](https://docs.github.com/en/actions/us
 We define GitHub Actions work under the [`.github`](https://github.com/software-gardening/almanac/tree/main/.github) directory.
 We suggest the use of [`act`](https://github.com/nektos/act) to help test GitHub Actions work during development.
 
+### Releases
+
+We utilize [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) ("semver") in order to distinguish between major, minor, and patch releases.
+We publish source code by using [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) available [here](https://github.com/software-gardening/almanac/releases).
+We publish a related Python package through the [Python Packaging Index (PyPI)](https://pypi.org/) available [here](https://pypi.org/project/almanack/).
+
+#### Release Publishing Process
+
+Several manual and automated steps are involved with publishing Software Gardening Almanack releases.
+See below for an overview of how this works.
+
+Notes about [semantic version](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) (semver) specifications:
+Software Gardening Almanack version specifications are controlled through [`poetry-dynamic-versioning`](https://github.com/mtkennerly/poetry-dynamic-versioning) which leverages [`dunamai`](https://github.com/mtkennerly/dunamai) to create version data based on [git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and commits.
+Software Gardening Almanack release git tags are automatically applied through [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) and related inferred changes from [`release-drafter`](https://github.com/release-drafter/release-drafter).
```

Comment:
```suggestion
## Version specifications 

- [`poetry-dynamic-versioning`](https://github.com/mtkennerly/poetry-dynamic-versioning) leveraging [`dunamai`](https://github.com/mtkennerly/dunamai) creates version data based on [git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) and commits.
- [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) automatically create git tags
- [`release-drafter`](https://github.com/release-drafter/release-drafter) infers and describes changes since last release
```

Please double check (and/or ignore) my edits here. I switched to active voice and tried to add a bit of structure for increasing clarity

Reply:
Thanks for this, looks a lot better!